sphinx/builders/linkcheck.py

```diff
@@ -158,14 +172,14 @@ def process_result(self, result: CheckResult) -> None:
                     red('broken    ') + result.uri + red(' - ' + result.message)
                 )
             self.write_entry(
-                'broken',
+                result.status,
```

Comment:
```suggestion
                LinkStatus.BROKEN.value,
```

Reply:
Using a specific symbol/status-code per `write_entry` call makes sense to me; I'm less sure about the `.value` access though.  What's the thinking there?

Reply:
`'broken'` is a literal, the closest semantic replacement is the `LinkStatus.BROKEN.value` constant, rather than the `result.status` variable.